### DISCUSSION TITLE
Enable Rainbow indent guides for python

### DISCUSSION THREAD

**Discussion by arnonmoscona** [izhangzhihao/intellij-rainbow-brackets#2635] [Q&A] [answered]
Your issue may already be reported!
Please search on the [issues](https://github.com/izhangzhihao/intellij-rainbow-brackets/issues) and the [document](https://github.com/izhangzhihao/intellij-rainbow-brackets/blob/2020.3/README.md) and [discussions](https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions) before creating one.

## Please check

- [x] I already support this project.
- [x] I already paid for this plugin.
- [ ] I'm on a trial of this plugin.
- [x] This issue/feature request is not reported before.

## Your programming languages
Python, HTML, SQL, XML

## Expected Behavior
* When editing Python code, the indent guides got the rainbow treatments before. After the last update, they stopped.

## Current Behavior
* No rainbow indent guides, even though the feature is turned on in the preferences and the plugin is licensed.

## Possible Solution
???

## Code snippet for reproduce (for bugs)
Please provide code snippet for reproduce bugs.

## Your Environment

* Plugin version: 2023.2.7
* IDE & Operating System version, comment your env as below(go to "About IntelliJ IDEA" -> click the "copy" icon):
* NOTE: If you are going to report a bug but WITHOUT your env information, your issue might be closed directly.

```
IntelliJ IDEA 2023.1 (Ultimate Edition)
Build #IU-231.8109.175, built on March 28, 2023
Licensed to Arnon Moscona
Subscription is active until December 9, 2023.
Runtime version: 17.0.6+10-b829.5 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.3.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 16
Metal Rendering is ON
Registry:
    analyze.exceptions.on.the.fly=true
    documentation.show.toolbar=true
    editor.minimap.enabled=true

Non-Bundled Plugins:
    com.github.leomillon.uuidgenerator (4.5.1)
    Quick Notes (3.3)
    com.intellij.javafx (1.0.4)
    lermitage.intellij.extratci (1.25.0)
    bundled-intellij-idea-help (222.4554.10)
    com.github.mfedko.idea.plugins.filelanguage (0.1.1)
    org.plugin.dot.id (1.4.1)
    com.sharedvocabs.plugins.rdfandsparql (231.911)
    Key Promoter X (2023.1.0)
    com.tabnine.TabNine (1.0.11)
    com.intellij.jira (4.4.2)
    launch-url-from-string (1.0)
    com.euphoricity.gitignore (0.3)
    Jira Browser (0.3.3)
    String Manipulation (9.7.1)
    GrepConsole (12.17.211.6693.0)
    Pythonid (231.8109.175)
    net.ashald.envfile (3.4.1)
    com.uriyyo.evaluate_async_code (1.22)
    com.koxudaxi.pydantic (0.4.2-231)
    com.github.shiraji.findpullrequest (1.7.2)
    net.seesharpsoft.intellij.plugins.csv (3.2.0-231)
    izhangzhihao.rainbow.brackets (2023.2.7)
    com.intellij.bigdatatools (231.8109.175)
    ru.adelf.idea.dotenv (2023.1)
    zielu.gittoolbox (500.0.8+213)
    mobi.hsz.idea.gitignore (4.5.0)
    aws.toolkit (1.67-231)
    com.intellij.plugins.html.instantEditing (231.8109.91)

Kotlin: 231-1.8.20-IJ8109.175
```

[a comment by an automated account is not shown]

**Comment by izhangzhihao** [comment]
- Can you try this to check out your license status? https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2630
- Can you provide any screenshots?
- Please show me your config screenshot, so I could try to help

**Comment by izhangzhihao** [comment]
Since there are no updates here, I will convert this issue to a discussion. Please continue to update there if your case is still not resolved.

**Comment by EfourC** [comment]
I hadn't been able to get it working for me either (using Python), and then I just realized that it **_partially_** works, but not as displayed in the Python example images in the readme. I haven't tried any other languages.

It doesn't work for all Python scopes, but it does seem to work for multi-line indents that are inside of brackets:

![z5QGAeetm4](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/121468033/f51a6d64-8907-40b3-b7d5-5e60e985a5c1)

The behavior is the same whether `Rainbowify Python keywords` is enabled or disabled.

![pycharm64_r8fsq38W2E](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/121468033/bb1bd1f5-2507-4d65-acce-2ebd366b02c1)

**Reply by izhangzhihao** [reply]
Your screenshot shows that this plugin is not activated yet, you need active this plugin for full support of rainbow python ident guidelines. You can try a 30 days free trial.

**Comment by izhangzhihao** [comment, marked as answer]
Your screenshot shows that this plugin is not activated yet, you need active this plugin for full support of rainbow python ident guidelines. You can try a 30 days free trial.

**Reply by EfourC** [reply]
Good to know, thanks!
